### DISCUSSION TITLE
animate svg logo

### DISCUSSION THREAD

**Discussion by chrisatbest** [senshu/Sozi#711] [General]
Is there a way of including everything in one single SVG package like with an animated gif ?

**Comment by ejvindh** [comment]
This was how the previous version of Sozi worked. It was a plugin for the Inkscape editor, and the resulting file was an SVG with interactive capabilities. I beleive 13.11 was the last version of Sozi that worked that way:
https://sozi.baierouge.fr/release-13.11-stable.html
...but I think you will find it hard to get it working today.

I think it should be possible to extract a SVG-file out of the current html-files that are created. That would, however, demand knowledge that I don't have.
